ios/Handlers/RNLongPressHandler.m

```diff
@@ -24,12 +28,21 @@ @implementation RNBetterLongPressGestureRecognizer {
 
 - (id)initWithGestureHandler:(RNGestureHandler*)gestureHandler
 {
-  if ((self = [super initWithTarget:gestureHandler action:@selector(handleGesture:)])) {
+  if ((self = [super initWithTarget:self action:@selector(handle:)])) {
     _gestureHandler = gestureHandler;
   }
   return self;
 }
 
+- (void)handle:(UIGestureRecognizer *)recognizer
+{
+  if (recognizer.state == UIGestureRecognizerStateBegan) {
+    startTime = [NSDate timeIntervalSinceReferenceDate];
```

Comment:
You probably want to use [`mach_absolutetime`](https://stackoverflow.com/a/30363702) here.

Reply:
I have changed how the duration is calculated and now it is updated when the state changes and I have replaced `timeIntervalSinceReferenceDate` with `mach_absolute_time`.